### ISSUE TITLE
Escape new lines in JSON returned value (here creating features)

### ISSUE THREAD

**Issue by zengbo0710** [ff4j/ff4j#451]
Create a Feature issue if the description has a new line in version 1.8.7

Response like below, if the description has a new line.
```json
{"uid":"Test","enable":false,"description":"Test
Test","group":null,"permissions":[],"flippingStrategy":null,"customProperties":{}}
```

description example:
```
Test
Test
```

**Comment by clun** [comment]
You are here using the REST API. I reproduce the error. Solution is to escape new lines in JSON strings.

**Comment by zengbo0710** [comment]
> You are here using the REST API. I reproduce the error. Solution is to escape new lines in JSON strings.
> 
> Are you using the spring-boot-starter of the `ff4j-webapi-jersey2x` api implementation ?

I didn't use `ff4j-webapi-jersey2x`.

**Comment by zengbo0710** [comment]
Create Feature
![image](https://user-images.githubusercontent.com/2897576/94993537-5dc73f80-05c4-11eb-8092-fcca8caff557.png)

Feature details gone
![image](https://user-images.githubusercontent.com/2897576/94993549-6a4b9800-05c4-11eb-8573-8103f9d8aaf1.png)

**Comment by Sleepy78** [comment]
Hi :)
I add +1 to @zengbo0710
I think this error was already raised and mentionned on this ticket https://github.com/ff4j/ff4j/issues/341 with fix suggestion.
Hope you can fix it because it is very unconfortable at usage.
Users are seeing a multiline field so they naturally think they can add new line. If they do, the feature is not editable anymore..

Best regards

**Comment by clun** [comment]
Issue will be fixed with `1.9` (release 20/12/2022)

<img width="592" alt="Screenshot 2022-12-19 at 22 58 50" src="https://user-images.githubusercontent.com/726536/208532242-be16060c-a320-489e-b63e-d7dfd0dbd603.png">
<img width="347" alt="Screenshot 2022-12-19 at 22 59 00" src="https://user-images.githubusercontent.com/726536/208532246-f42e0a95-23be-4693-b471-dba9ef38ccb5.png">
<img width="603" alt="Screenshot 2022-12-19 at 22 59 09" src="https://user-images.githubusercontent.com/726536/208532255-2802d0d8-b3a2-4fc8-93d7-1196a2c04c1e.png">
